### ISSUE TITLE
Add disablePictureInPicture property to HTML5 Video Elements

### ISSUE THREAD

**Issue by eek** [microsoft/TypeScript#36324]
## Search Terms

disablePictureInPicture, "picture-in-picture"

## Suggestion

Add suppport for `disablePictureInPicture` on HTML5 Video Elements.

## Use Cases & Examples

Remove More Button on Video Elements and not give an error.

Examples of usage for the property:

https://github.com/facebook/react/issues/15333

Currently it give the error: 

```
TS2339 Property 'disablePictureInPicture' does not exist on HTML5VideoElement.
```

Which is supported in all Chromium-based browsers (by adding this tag to the HTML5 Video Element, you remove the Picture in Picture option from the Video Element's Menu.

## Checklist

My suggestion meets these guidelines:

* [x] This wouldn't be a breaking change in existing TypeScript/JavaScript code
* [x] This wouldn't change the runtime behavior of existing JavaScript code
* [x] This could be implemented without emitting different JS based on the types of the expressions
* [x] This isn't a runtime feature (e.g. library functionality, non-ECMAScript syntax with JavaScript output, etc.)
* [x] This feature would agree with the rest of [TypeScript's Design Goals](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Design-Goals).

**Comment by MartinJohns** [comment]
Should experimental features really be added to the default library?

**Comment by jakebailey** [comment]
This was fixed in #44684.
